third_party/ObjexxFCL/src/ObjexxFCL/Array1S.hh

```diff
@@ -213,6 +213,18 @@ public: // Assignment: Array
 
 public: // Assignment: Value
 
+	template< typename U, Size s, class = typename std::enable_if< std::is_assignable< T&, U >::value >::type >
```

Comment:
Added template for assigning `std::array` to `Array1S` (i.e., `Array1` slices of higher dimensional `ArrayXD`).

Reply:
Not gonna lie, it's a little offputting to see more being added to Array1D after we've spent so much time stripping it down.  Nothing red flag, just surprising.  I wonder if this method was already available in the past and it wasn't used so it was removed at some point.

Reply:
I get it. I wasn't happy about this either. I added it so that built-in glycols and user-defined glycols could be handled with the same code. The real solution is to just convert this entire module to `std::vector` which is eminently doable since it is quite self contained, but just something that I didn't feel like doing in this PR. I do have another pass planned having to do with caching, so I might do it then.